### PR TITLE
V1.0 tweaks

### PR DESCRIPTION
**Pull request by Xuan127**

### CODE REVIEW

File: src/test/java/cashleh/ParserTest.java
Comment (danielpappa) [inline src/test/java/cashleh/ParserTest.java:1]: Very good JUnit testing! Implements all necessary test cases so far for the parser class, well done!

File: src/test/java/cashleh/ParserTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    IncomeStatement incomeStatement = new IncomeStatement();
+    Parser parser = new Parser(new ExpenseStatement(), new IncomeStatement());
+
+    @Test
+    public void parserUnknownCommandTest() {
+        String inputString = "test";
+        try {
+            assertEquals(parser.parse(inputString), null);
+        } catch (CashLehException e) {
+            assertEquals("Aiyoh! Your input blur like sotong... Clean your input for CashLeh!", e.getMessage());
```
Comment (danielpappa) [inline src/test/java/cashleh/ParserTest.java:30]: I think I already commented on this line in the Parser class, but I believe we should not use slang/singlish in our code's answers (even though that requirement could also only be interpreted as being limited to actual code and not string outputs...)

**Review by danielpappa — APPROVED** [review]

File: src/test/java/cashleh/ParserTest.java (hunk 1 above)
Comment (BenedictChannn) [reply src/test/java/cashleh/ParserTest.java:30]: I think it's fine since the messages are fit for our context.

**Review by BenedictChannn — APPROVED** [review]
Looks good! But we can probably look to add more tests like invalid inputs too!

**Pull request merged**